tools/toolchains/__init__.py

```diff
@@ -344,7 +344,7 @@ def get_symbols(self):
 
             # Config support
             if self.has_config:
-                self.symbols.append('HAVE_MBED_CONFIG_H')
+                self.symbols.append('HAVE_MBED_CONFIG')
```

Comment:
But currently it doesn't, so I guess this is meant to be implemented later. In which case I'd leave a comment explaining this in the source, because otherwise we're likely to forget this later.